BloomFilter.md

```diff
@@ -181,6 +182,9 @@ struct ColumnMetaData {
 
 ```
 
+The Bloom filter data is stored right after pages indexes, the file layout is look like:
```

Comment:
```suggestion
The Bloom filter data is stored right after the page indexes, and the file layout looks like:
```

Reply:
This is still incorrect. Can you look at my suggestion more carefully and identify what you missed?

Reply:
Oops.

Reply:
@jbapple  @chenjunjiedada  Hi for some reason this statement has now says "before the page indexes" but the images has the bloom filter after the pages indices. Should be this be updated?